e2e/tests/localAuthoritySubmitsApplication_test.js

```diff
@@ -426,6 +426,13 @@ Scenario('local authority enters attending hearing',async (I, caseViewPage, ente
 
 Scenario('local authority uploads documents', caseDocs.uploadDocuments());
```

Comment:
this line seems like it is in the wrong place now?

Reply:
In what way?

Reply:
Made tech debt ticket to refine and improve readability of the caseDocuments fragment